sql/upgrade/1.3.3_schema/postgresql/dolphinscheduler_ddl.sql

```diff
@@ -79,4 +80,9 @@ d//
 
 delimiter ;
 SELECT ct_dolphin_T_t_ds_process_definition_version();
-DROP FUNCTION IF EXISTS ct_dolphin_T_t_ds_process_definition_version();
\ No newline at end of file
+DROP FUNCTION IF EXISTS ct_dolphin_T_t_ds_process_definition_version();
+
+-- add t_ds_resources_un
+ALTER TABLE t_ds_resources ADD CONSTRAINT t_ds_resources_un UNIQUE (full_name,"type");
```

Comment:
Hi,This place need judge whether t_ds_resources_un exist,you can use the process function like the above uc_dolphin_T_t_ds_task_instance_A_first_submit_time